azure-stack/hci/manage/manage-bitlocker.md

```diff
@@ -73,9 +86,9 @@ Follow these steps to disable BitLocker encryption:
     Disable-ASBitLocker
     ```
 
-## Get BitLocker recovery keys
+## Get recovery keys for volume encryption with BitLocker
```

Comment:
Same as above, please leave the old and dont rename to volume encryption with BL...